src/main/java/net/minecraftforge/fml/client/config/GuiConfigEntries.java

```diff
@@ -1665,6 +1665,42 @@ public String getName()
         @Override
         public void onGuiClosed()
         {}
+
+        /**
+         * Get string surrounding tagged area.
+         */
+        private String removeTag(String target, String tagStart, String tagEnd)
+        {
+            String taglessResult = "";
+            int tagStartPosition = -1;
+            int tagEndPosition = -1;
+
+            for (int i = 0; i < target.length(); i++)
```

Comment:
This section seems very complicated. Can't you use something like `String.indexOf(String)`?